manifests/init.pp

```diff
@@ -381,6 +406,11 @@
   Optional[Variant[Numeric, String[1]]] $package_apt_pin                                           = undef,
   String $package_ensure                                                                           = 'installed',
   Optional[String] $package_gpg_key                                                                = undef,
+  Optional[Boolean] $quorum_membership_reconciliation_enabled                                      = undef,
```

Comment:
Ok, so we're leaving this one optional still? I'm Ok with it, just confirming 👍

Reply:
I think optional makes sense to me, this allows it to be "undef" by default, and therefore truly "unset" completely, and totally, unless you opt to enable it one way or another. Forcing it to false in my mind implies I'm "forcing it to false" - but since i cannot do that on RabbitMQ 3.12 and lower, undef leaves it completely unset and unmanaged, unless I set it.